# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bemenu/fpic.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

force -fPIC globaly as its missing somewhere (see github PR)
--- bemenu-0.5.0/GNUmakefile.vanilla    2021-05-11 00:59:36.016900287 +0200
+++ bemenu-0.5.0/GNUmakefile    2021-05-11 00:56:41.279892021 +0200
@@ -12,7 +12,7 @@
        -Wfloat-equal -Wcast-align -Wpointer-arith -Wchar-subscripts -Warray-bounds=2 -Wno-unknown-warning-option
 
 override CFLAGS ?= -g -O2 $(WARNINGS)
-override CFLAGS += -std=c99
+override CFLAGS += -std=c99 -fPIC
 override CPPFLAGS ?= -D_FORTIFY_SOURCE=2
 override CPPFLAGS += -DBM_VERSION=\"$(VERSION)\" -DBM_PLUGIN_VERSION=\"$(VERSION)-$(GIT_SHA1)\" -DINSTALL_LIBDIR=\"$(PREFIX)$(libdir)\"
 override CPPFLAGS += -D_DEFAULT_SOURCE -Ilib
